### DISCUSSION TITLE
MkDocs fediverse group

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocs/mkdocs#3246] [Ideas]
What do you think about setting up a group for MkDocs on the fediverse? See https://fosstodon.org/@feditips@mstdn.social/110498250226529056:

> There's a really good community feature on here called groups. It works like this:
>
> 1. Follow a group's account
> 2. You will start seeing posts on that topic
> 3. You can post to the group by @-ing it
>
> It's more powerful than hashtag following, as all group posts are actively pushed to all of the group's members. No one misses anything.
>
> Find out more about groups at https://fedi.tips/how-to-use-groups-on-the-fediverse/
>
> If you're interested, try following one of the friendly active groups curated at https://fedi.directory/tag/fediverse-groups/

**Comment by Andre601** [comment]
Could be interesting.
I myself made an account on chirp.social to allow people share their mc modding stuff.

It's neat.

**Reply by pawamoy** [reply]
I created the group, see https://fosstodon.org/@pawamoy/110586618469898174. Let see how that goes :man_shrugging:

**Reply by pawamoy** [reply]
By the way I can add other managers to the group, just share your email with me if you're interested (direct message on twitter/fosstodon, or publicly here if you don't mind) :)
